lib/fetch/index.js

```diff
@@ -120,137 +120,151 @@ class Fetch extends EE {
   }
 }
 
+let tracingChannel
+try {
+  const diagnosticsChannel = require('diagnostics_channel')
+  tracingChannel = diagnosticsChannel.tracingChannel('undici')
+} catch {
+  tracingChannel.start = { hasSubscribers: false }
+  tracingChannel.end = { hasSubscribers: false }
+  tracingChannel.asyncStart = { hasSubscribers: false }
+  tracingChannel.asyncEnd = { hasSubscribers: false }
+  tracingChannel.error = { hasSubscribers: false }
```

Comment:
We don't need this try/catch because fetch is only supported in node >= 16.8.0. Diagnostics_channel was added in some version of node 14.

Reply:
just updated the PR to remove try/catch on DC